src/Core/src/Handlers/Image/ImageHandler.Android.cs

```diff
@@ -0,0 +1,41 @@
+﻿#nullable enable
+using System.Threading.Tasks;
+using Android.Widget;
+using AndroidX.AppCompat.Widget;
+
+namespace Microsoft.Maui.Handlers
+{
+	public partial class ImageHandler : ViewHandler<IImage, ImageView>
+	{
+		protected override ImageView CreateNativeView() => new AppCompatImageView(Context);
+
+		protected override void DisconnectHandler(ImageView nativeView)
+		{
+			base.DisconnectHandler(nativeView);
+
+			_sourceManager.Reset();
+		}
+
+		public static void MapAspect(ImageHandler handler, IImage image) =>
+			handler.NativeView?.UpdateAspect(image);
+
+		public static void MapIsAnimationPlaying(ImageHandler handler, IImage image) =>
+			handler.NativeView?.UpdateIsAnimationPlaying(image);
+
+		public static void MapSource(ImageHandler handler, IImage image) =>
+			MapSourceAsync(handler, image).FireAndForget(handler);
+
+		public static async Task MapSourceAsync(ImageHandler handler, IImage image)
+		{
```

Comment:
Should we include `CancellationToken` here to allow cancel the loading operation?

Reply:
This happens inside:

```csharp
var token = handler._sourceManager.BeginLoad();
```

The existing token is cancelled and starts a new source and returns the token to be used with `UpdateSourceAsync`